src/styles/companies.css

```diff
@@ -14,11 +14,22 @@
   padding: 5px;
 }
 
+.header {
+  display: flex;
+  justify-content: space-around;
+}
+
 .loading {
   font-family: "Lato", sans-serif;
   font-style: italic;
 }
 
+.back {
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+  font-size: 25px;
+}
+
 .search {
   display: flex;
   gap: 10px;
```

Comment:
- Hey, great job adding the icons, however, the date is no longer in the right position, the icons are not well aligned, and also the < icon is no longer showing as expected.
- You can reduce the width of the search bar and remove the text `search` to help make all the icons show as expected.



![Screenshot from 2023-02-10 22-29-49](https://user-images.githubusercontent.com/59397159/218202973-bc12ca54-e135-448e-b6b5-12ae5973e486.png)

Reply:
Thanks for that, I have worked on it now 💯 💯 💯 ...!